ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py

```diff
@@ -12,6 +10,49 @@
 from ivy_tests.test_ivy.helpers import handle_frontend_test
 
 
+# nanmedian
+@handle_frontend_test(
+    fn_tree="numpy.nanmedian",
+    dtype_and_a=statistical_dtype_values(function="median"),
+    dtype=helpers.get_dtypes("float", full=False, none=True),
+    where=np_frontend_helpers.where(),
+    keep_dims=st.booleans(),
+)
+def test_numpy_nanmedian(
+    dtype_and_a,
+    dtype,
+    where,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+    keep_dims,
+):
+    input_dtype, a, axis = dtype_and_a
+    if isinstance(axis, tuple):
+        axis = axis[0]
+    where = np_frontend_helpers.handle_where_and_array_bools(
+        where=where,
+        input_dtype=input_dtype,
+        as_variable=test_flags.as_variable,
```

Comment:
you seem to misunderstand the errorlog here
the function does not need this "as_variable=test_flags.as_variable," line
so I recommend you to consider carefully about the purpose of this "handle_where_and_array_bools" and how to use it by checking its implementation.

Reply:
where can I find its implementation?

Reply:
ivy_tests/test_ivy/test_frontends/test_numpy/helpers.py

Reply:
As I understood:  np_frontend_helpers.handle_where_and_array_bools returns (where, input_dtypes, test_flags) to assign the values like that as done in other test functions.
        where=where,
        input_dtype=input_dtypes,
        test_flags=test_flags,
    
    I think that the errors result from problems in the helper functions themselves, not my test function. what is your opinion? what should I do to merge this PR? 
    
    thank you very much for your time